website/docs/language/resources/provisioners/syntax.mdx

```diff
@@ -234,10 +234,7 @@ fail, Terraform will error and rerun the provisioners again on the next
 `terraform apply`. Due to this behavior, care should be taken for destroy
 provisioners to be safe to run multiple times.
 
-```
-Destroy provisioners of this resource do not run if `create_before_destroy`
-is set to `true`. This [GitHub issue](https://github.com/hashicorp/terraform/issues/13549) contains more details.
-```
+~> **NOTE:** Destroy provisioners of this resource do not run if `create_before_destroy` is set to `true`. This [GitHub issue](https://github.com/hashicorp/terraform/issues/13549) contains more details.
```

Comment:
```suggestion
~> **Note**: Destroy provisioners of this resource do not run if `create_before_destroy` is set to `true`. This [GitHub issue](https://github.com/hashicorp/terraform/issues/13549) contains more details.
```

Reply:
Hello, @irth! 👋  I apologize for the late review here, but thank you for catching this! In the interest of getting this in quickly, I made a small suggestion for style above and am going to commit it, so I can review the deploy preview and get this merged asap!